google/cloud/firestore_v1/rate_limiter.py

```diff
@@ -17,7 +17,7 @@
 
 
 def utcnow():
-    return datetime.datetime.utcnow()
+    return datetime.datetime.now(tz=datetime.timezone.utc)
```

Comment:
Note that this changes a public interface. I'm guessing it won't break anyone if people are consistently doing things like:

```python
dt_end = rate_limiter.utcnow() + timedelta(seconds=30)
while rate_limiter.utcnow() < dt_end:
    sleep(1)
```

But if they are mixing between `rate_limiter.utcnow()` and `datetime.utcnow()` or some other thing that generates a naïve datetime, this will cause `TypeError`s.

(I'm also not familiar with this library's conventions, so maybe this isn't really a public interface anyway).

On the plus side, it seems like some of [the other places in the GCP framework](https://github.com/googleapis/python-cloud-core/blob/44de0d893daab18b74de9aa032f7e580a2e47d22/google/cloud/_helpers/__init__.py#L190) are treating naïve datetimes as UTC anyway, so that will probably reduce the blast radius here.

Reply:
Deprecated `google.cloud.firestore_v1.rate_limiter.utcnow` in 075e999a4638fd137e65757f8d5a65a3aa3fcef9